landscape/package/reporter.py

```diff
@@ -28,6 +29,9 @@
 HASH_ID_REQUEST_TIMEOUT = 7200
 MAX_UNKNOWN_HASHES_PER_REQUEST = 500
 LOCK_RETRY_DELAYS = [0, 20, 40]
+PYTHON_BIN = "/usr/bin/python3"
```

Comment:
Will this be true even for older versions of ubuntu, like trusty? That the python interpreter will be python3?

Reply:
I did the testing starting from Trusty 14.04 (the oldest version, without suffixes) and python3 is already installed there (as a dependency of ubuntu-minimal) and after running do-release-upgrade the release-upgrader that has been downloaded was using python3.
Since that's true for the oldest supported release I thought it's a safe assumption then.